src/ConnectedMode.UnitTests/Migration/MSBuildFileCleanerTests.cs

```diff
@@ -37,5 +41,68 @@ public void MefCtor_CheckIsExported()
         [TestMethod]
         public void MefCtor_CheckTypeIsNonShared()
             => MefTestHelpers.CheckIsNonSharedMefComponent<MSBuildFileCleaner>();
+
+        [TestMethod]
+        public void Clean_EmptyDocument_NoErrors_ReturnsNull()
+        {
+            const string content = "<Project/>";
```

Comment:
The input content must be valid XML, so it can't be `null` or empty (if the file is not valid XML we won't get this far).